tools/pip_install.sh

```diff
@@ -74,10 +74,18 @@ guid_generator_setup()
   cd ..
 }
 
-
+abel_setup()
+{
+  echo "Starting ABEL setup"
+  cd abel
+  python -m pip install .
+  echo "Finished ABEL setup"
+  cd ..
+}
 
 ontology_validator_setup
 instance_validator_setup
```

Comment:
todo: we should probably revisit those? have the user select what they want to install?

Reply:
Since these files have already been landed, I'll open another bug to address this functionality.